test/test_proxy_tensor.py

```diff
@@ -1269,6 +1296,10 @@ def f(a):
         # dtype inference is correct
         self.assertExpectedInline(r, """\
 def forward(self, a_1):
+    sym_size_int = torch.ops.aten.sym_size.int(a_1, 0);  a_1 = None
+    sym_float = torch.sym_float(sym_size_int);  sym_size_int = None
+    ge = sym_float >= 0.0
+    pow_1 = sym_float ** 2.0;  sym_float = None
```

Comment:
torch.tensor

Reply:
The compute here is kind of unfortunate because it ends up specializing, c'est la view (Dynamo will do the right thing; eager COULD do the right thing if we make the torch.tensor constructor smarter)